moment/moment.d.ts

```diff
@@ -276,6 +277,12 @@ declare module moment {
         isSame(b: Date, granularity: string): boolean;
         isSame(b: number[], granularity: string): boolean;
 
+        isBetween(a: Moment, b: Moment, granularity?: string): boolean;
```

Comment:
The [docs](http://momentjs.com/docs/#/query/is-between/) say the first two parameters should be moment-like which means this doesn't work `moment().isBetween(moment(), new Date())`. You'd need to use union types to make it match the docs.
```ts
isBetween(a: Moment|string|number|Date|number[], b: Moment|string|number|Date|number[], granularity?: string): boolean;
```